event-store-management/event-store-management-core/src/main/java/uk/gov/justice/services/eventstore/management/aggregate/snapshot/regeneration/commands/RebuildSnapshotCommand.java

```diff
@@ -5,7 +5,9 @@
 public class RebuildSnapshotCommand extends BaseSystemCommand {
 
     public static final String REBUILD_SNAPSHOTS = "REBUILD_SNAPSHOTS";
-    private static final String DESCRIPTION = "Forces the generation of a new aggregate snapshot for a given stream id";
+    private static final String COMMAND_RUNTIME_ID_TYPE = "streamId";
+    private static final String COMMAND_RUNTIME_STRING_TYPE = "aggregate class name";
+    private static final String DESCRIPTION = "Forces the generation of a new aggregate snapshot for a given " + COMMAND_RUNTIME_ID_TYPE + " and " + COMMAND_RUNTIME_STRING_TYPE;
```

Comment:
Note sure we should use the word "Force" as we do not overwrite. We generate only if there is no one or if the existing one cannot be de-serialized